Sources/Stevenson/JiraService+FieldTypes.swift

```diff
@@ -0,0 +1,86 @@
+import Vapor
+
+// MARK: Jira Common Field Types
```

Comment:
This is only an extraction of [this `JiraService.FieldType` enum](https://github.com/babylonhealth/Stevenson/pull/43/files#diff-ff1a5159f3d69ff0886fa943d9447375L112) in a dedicated file. The content of this `extension` was not changed, just cut/paste.